master-qa-15030

simonparksays opened the pull request:
https://issues.liferay.com/browse/LIFERAYQA-15030

LeventeHudak committed:
LPS-54611 Be able to generate upgrade tables from the modules using the same process

LeventeHudak committed:
LPS-54611 Adding ant target for building upgrade tables from bookmarks module

Commit:
LPS-54611 SF

Commit:
LPS-54851 Project creation

Commit:
LPS-54851 Move resource from portal-web

Commit:
LPS-54851 Create MVCPortlet

Commit:
LPS-54851 Remove needless configuration

Commit:
LPS-54851 Create user notification handler

Commit:
LPS-54851 Create control panel entry

Commit:
LPS-54851 Create action commands

Commit:
LPS-54851 Modify jsp pages to work on OSGi bundle

Commit:
LPS-54851 Localization stuff

Commit:
LPS-54851 OSGi porting done

Commit:
LPS-54851 Refactoring jsp pages

Commit:
LPS-54851 SF

Commit:
LPS-54851 Extract constants

Commit:
LPS-54851 Refactoring

Commit:
LPS-54851 SF

Commit:
LPS-55497 project files

Commit:
LPS-55497 move resources

Commit:
LPS-55497 migrate portlet

Commit:
LPS-55497 move action commands

Commit:
LPS-55497 move staged models

Commit:
LPS-55497 modify jsp pages

Commit:
LPS-55497 language files

liferay-continuous-integration commented:
<h1>All tests PASSED!</h1><p>Build Time: 20 minutes 30 seconds</p><h1>See the following links for more information.</h1><ul><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest(master)/848/">https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest(master)/848/</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-start(master)/848/">https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-start(master)/848/</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/841/">https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/841/</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-portalweb(master)/122/">https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-portalweb(master)/122/</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-stop(master)/840/">https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-stop(master)/840/</a></li></ul>

Commit:
LPS-54553 - Breadcrumb overlaps other portlets when scrolling in narrow view

sharonchoi commented:
Pushed tests back to SPark:
-please break up commits into smaller, more modular/logical chunks, it's ok to mix up paths w/ macros
-please add line breaks according to new standard here: https://in.liferay.com/web/global.engineering/wiki/-/wiki/Quality+Assurance+Main/POSHI+Line+Spacing+Rules

Commit:
LPS-55497 modify references to ddm resources

Commit:
LPS-55497 resource-actions

Commit:
LPS-55497 Auto SF

Commit:
LPS-55749 move javascript method to onclick method

jonathanmccann committed:
LPS-55621 Test

CAustin582 committed:
LPS-54611

Ithildir committed:
LPS-54611 Project files

Ithildir committed:
LPS-54611 Move

Ithildir committed:
LPS-54611 SF

Commit:
LRQA-15030 Update Blogs macros that add entry content for revamp

marcellustavares committed:
LPS-55497 SF

Commit:
LRQA-15030 Update Blogs macros that create entry drafts

marcellustavares committed:
LPS-55497 Rename

Commit:
LRQA-15030 Update Blogs macros used for navigation

Commit:
LRQA-15030 Update Blogs macros that edit entry content

Commit:
LRQA-15030 Update Blogs macros that edit the entry settings tab

marcellustavares committed:
LPS-55497 Update references

marcellustavares committed:
LPS-55497 Move ADT to DDM for now b/c currently they share the same resources

Commit:
LRQA-15030 Update Blogs macros that subscribe and edit permissions

Commit:
LRQA-15030 Update Blogs macros for Aggregator/Recent Authors portlet

marcellustavares committed:
LPS-55497 Init Portlet Display Template module

marcellustavares committed:
LPS-55497 Move resources

Commit:
LRQA-15030 Update Blogs macros that use workflow and Asset Publisher

Ithildir committed:
LPS-54611 Remove dependencies

marcellustavares committed:
LPS-55497 This does not work, made a small change to make DL file entry types working again

marcellustavares committed:
LPS-55497 Register portlet display template on the necessary actions

Ithildir committed:
LPS-54611 Use module

marcellustavares committed:
LPS-55497 Make getDDMForm operation accept portletRequest

marcellustavares committed:
LPS-55497 Update resource paths

tomwang2011 committed:
LPS-55780 Create third-party module

marcellustavares committed:
LPS-55497 Espose inner beans

sharonchoi commented:
SF for SPark:
-removed duplicate locator key for permissions checkbox in asset publisher

sharonchoi commented:
Pushed to BChiu:
https://github.com/brianchiu/liferay-portal/pull/293

marcellustavares committed:
LPS-55497 Fix test

marcellustavares committed:
LPS-55497 SF

marcellustavares committed:
LPS-55497 Sort and add ADT portlet title

marcellustavares committed:
LPS-55497 build lang

Ithildir committed:
LPS-54611 Simpler

Ithildir committed:
LPS-54611 Fix copyright file retrieval

Ithildir committed:
LPS-54611 Pass settings to SDK from portal

shuyangzhou committed:
LPS-55630 Remove the dead code added by LEP-1250 515c1a23c1d749cedc68b1bb27b8555f5b98ad72, portal-web never really have a docroot/WEB-INF/src

Ithildir committed:
LPS-54611 Adapt tool to be called from modules

tomwang2011 committed:
LPS-55780 update ivy dependency

Ithildir committed:
LPS-54611 Rename variable

hhuijser committed:
LPS-55690 Add check to SF

hhuijser committed:
LPS-55690 Add test

jkappler committed:
LPS-55704 RSS: Add display context to context objects and pass the value to the DDM Template Renderer

jkappler committed:
LPS-55704 RSS: Add the display context to the palette

jkappler committed:
LPS-55704 RSS: Use configuration values in the template instead of preferences

Commit:
LPS-55822 Make methods public

jkappler committed:
LPS-55704 Categories Navigation: Add display context to context objects and pass the value to the DDM Template Renderer

jkappler committed:
LPS-55704 Categories Navigation: Add the display context to the palette

jkappler committed:
LPS-55704 Site Map: Add display context to context objects and pass the value to the DDM Template Renderer

jkappler committed:
LPS-55704 Site Map: Add the display context to the palette

Commit:
LPS-55822 Allow extending sitemap URLs

Commit:
LPS-55822 Move logic to providers

IstvanD committed:
LPS-55619 File Entry still has the old value of it's version field after reverting

marcellustavares committed:
LPS-54851 Move packages

marcellustavares committed:
LPS-54851 Move displayContext initialization to jsp and rename variable

marcellustavares committed:
LPS-54851 SF

Commit:
LPS-53790 Remove fallback when default template is not available

This code would cause the first available template for this portlet to be rendered, without any control. Only a template selected by the admin or the default template defined by the portlet developer should be rendered.

ealonso committed:
LPS-55836 Removes no needed struts paths

tomwang2011 committed:
LPS-55780 remove permissions file from jar

shuyangzhou committed:
LPS-55630 Marcodef portal-web manifest file generation

ealonso committed:
LPS-55836 Creates a new taglib for layout template list

shuyangzhou committed:
LPS-55630 No point to call super, just copy the jar file

willnewbury committed:
LPS-55428 add missing build.xml to control-panel app

shuyangzhou committed:
LPS-55630 Call jar-marco, rather than depending on the jar target

shuyangzhou committed:
LPS-55630 Refactor deploy-properties to be marcodef.

shuyangzhou committed:
LPS-55630 Apply usage of deploy-properties marcodef

shuyangzhou committed:
LPS-55630 Move deploy-properties target to build-common.xml so that we can use it everywhere, no need to force portal-impl/portal-web to redefine it

brianchandotcom committed:
LPS-55822 SF

Commit:
LPS-53802 - PortletToolbarTest - only wait for missing dependencies

brianchandotcom committed:
LPS-55704 sort by templateVariableGroup.add*Variable("keytosortby"

brianchandotcom committed:
LPS-55704 lang keys

robertoDiaz committed:
LPS-55794 Avoid undefined exception

brianchandotcom committed:
LPS-55704 follow the JSP pattern

brianwulbern committed:
LRQA-15810 Ignore LPS-55835 until it is fixed

brianchandotcom committed:
LPS-55621 SF

brianchandotcom committed:
LPS-55836 consistency with Java class name

shuyangzhou committed:
LPS-55630 Clean should by default removing deployed jar file

brianchandotcom committed:
LPS-55836 rename

shuyangzhou committed:
LPS-55630 Covered by clean-common already

shuyangzhou committed:
LPS-55630 Refactor clean to follow the new pattern

shuyangzhou committed:
LPS-55780 bnd

marcellustavares committed:
LPS-54851 Update references

marcellustavares committed:
LPS-54851 Fix SPA issue with links with preventDefault

shuyangzhou committed:
LPS-55780 build.xml

natecavanaugh committed:
LPS-55794 - Source formatting

brianchandotcom committed:
LPS-55837 sdk

brianchandotcom committed:
LPS-55837

brianchandotcom committed:
Record

brianchiu committed:
LRQA-15810 Make comment consistent with other comments

shuyangzhou committed:
LPS-55780 Update ivy dependency

marcellustavares committed:
LPS-55497 Move model resources back to portal for now to avoid initialization issues

Commit:
LPS-55550 update bnd to include antlr

marcellustavares committed:
LPS-55497 Not needed

natecavanaugh committed:
LPS-55560 - missed case

Commit:
LPS-55550 ensure portal and module classloaders are present

brianchandotcom committed:
LPS-53802 SF

brianchandotcom committed:
LPS-55635 sdk

brianchandotcom committed:
LPS-55635 test snapshots

brianchandotcom committed:
LPS-55635 delete this for now

brianchandotcom committed:
LPS-55822 @Override

brianchandotcom committed:
LPS-54611 not thrown

brianchandotcom committed:
LPS-54611 prep next version

brianchandotcom committed:
LPS-54611 sdk

brianchandotcom committed:
LPS-54611 ivy cache

brianchandotcom committed:
Record

brianchandotcom committed:
LPS-55550 move to BaseScriptingExecutor?

brianchandotcom committed:
LPS-55550 Move common logic

brianchandotcom committed:
LPS-55497 Sort

brianchandotcom committed:
LPS-55497 Sort

brianchandotcom committed:
LPS-55497 Sort

brianchandotcom committed:
LPS-55497 Sort

brianchandotcom committed:
LPS-55497 Sort

brianchandotcom committed:
LPS-55497 Sort

brianchandotcom committed:
LPS-55497 Sort

brianchandotcom committed:
LPS-55497 Sort

brianchandotcom committed:
LPS-55497 SF

brianchandotcom committed:
LPS-55497 Sort

brianchandotcom committed:
LPS-55497 Match edit_file_entry_type.jsp (search for fieldsJSONArrayString), and match edit_template_form.jsp

brianchandotcom committed:
LPS-55497 Sort

brianchandotcom committed:
LPS-55497 Sort

brianchandotcom committed:
LPS-54851 not needed

brianchandotcom committed:
LPS-54851 SF

brianchandotcom committed:
LPS-54851 Cleaner

brianchandotcom committed:
LPS-54851 not needed

brianchandotcom committed:
LPS-54851 SF

brianchandotcom committed:
LPS-54851 SF

brianchandotcom committed:
LPS-54851 consistency

brianchandotcom committed:
LPS-54851 lang keys

brianchandotcom committed:
LPS-54851 SF

brianchandotcom committed:
LPS-54851 SF

brianchandotcom committed:
LPS-54851 SF

brianchandotcom committed:
LPS-54851 method signature makes it obvious

brianchandotcom committed:
LPS-54851 auto SF

brianchandotcom committed:
LPS-54851 fix SF warning

brianchandotcom committed:
LPS-54851 SF

brianchandotcom committed:
LPS-54851 match searchTasksAssignedToMe

brianchandotcom committed:
LPS-54851 SF

brianchandotcom committed:
LPS-54851 consistency

brianchandotcom committed:
LPS-54851 SF

brianchandotcom committed:
LPS-54851 SF

brianchandotcom committed:
LPS-54851 as used

brianchandotcom committed:
LPS-54851 easier to read

brianchandotcom committed:
LPS-54851 consistency

brianchandotcom committed:
LPS-54851 consistency

brianchandotcom committed:
LPS-54851 auto SF

brianchandotcom committed:
LPS-54851 SF

brianchandotcom committed:
LPS-54851 SF

brianchandotcom committed:
LPS-54851 as used

brianchandotcom committed:
LPS-55780 use tstamp

brianchandotcom committed:
LPS-55780 missing file

brianchandotcom committed:
LPS-55780 SF

brianchandotcom committed:
LPS-55780 use the release

brianchandotcom committed:
LPS-55780 ivy cache

liferay-continuous-integration commented:
<h1>All tests PASSED!</h1><p>Build Time: 21 minutes 26 seconds</p><h1>See the following links for more information.</h1><ul><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest(master)/950/">https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest(master)/950/</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-start(master)/950/">https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-start(master)/950/</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/948/">https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/948/</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-portalweb(master)/111/">https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-portalweb(master)/111/</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-stop(master)/948/">https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-stop(master)/948/</a></li></ul>

Commit:
LRQA-15030 Remove conditionals from ViewDetails to create new macro

Commit:
LRQA-15030 Rename add to addEntry for clarity in blogs.macro

sharonchoi commented:
SF for SPark:
-removed duplicate locator key for permissions checkbox in asset publisher

sharonchoi commented:
Pushed to BChiu:
brianchiu#293
